sql/src/main/java/io/crate/operation/InputRow.java

```diff
@@ -48,4 +49,13 @@ public Object get(int index) {
     public Object[] materialize() {
         return Buckets.materialize(this);
     }
+
+    @Override
+    public String toString() {
```

Comment:
i guess this is for debugging. pls make sure that we do not print too much data in logs by using toString of the row